src/components/channel/ChannelCard.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+  Avatar,
+  AvatarGroup,
+  Badge,
+  Card,
+  CardBody,
+  CardHeader,
+  Heading,
+  Text,
+} from '@chakra-ui/react';
+import { formatTimeToKST } from '../../utils';
+import { Channel } from '../../@types/chat';
+
+interface Props {
+  channel: Channel;
+}
+
+const ChannelCard = ({ channel }: Props) => {
+  return (
+    <Card border="2px solid #cdcdcd" boxShadow={0} w={215}>
```

Comment:
아까 hover 어떻게 하냐구 디코에서 말씀하셨는데
```suggestion
    <Card border="2px solid #cdcdcd" boxShadow={0} w={215}  _hover={{
        bg: 'gray.300',
      }}>
       
```

이런 식으로 하시면 됩니다

Reply:
감사합니다~